admin-dev/filemanager/include/utils.php

```diff
@@ -103,7 +103,7 @@ function foldersize($path)
     $cleanPath = rtrim($path, '/'). '/';
 
     foreach ($files as $t) {
-        if ($t<>"." && $t<>"..") {
+        if ($t!="." && $t!="..") {
```

Comment:
can you please add spaces for readability?
```php
    if ($t != "." && $t != "..")
```

Reply:
Sure